packages/typo3-docs-theme/resources/public/css/theme.css

```diff
@@ -1,6 +1,6 @@
 @charset "UTF-8";
 /*!
- * Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com
+ * Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com
```

Comment:
Please run `npm ci` on the theme folder. In a previous PR I updated the node deps.

Reply:
this doesnt change the file. My version of fontawesome is higher then the one before for some reason

Reply:
ok, I hope I worked it out